app/src/main/java/org/oppia/android/app/home/recentlyplayed/PromotedStoryViewModel.kt

```diff
@@ -7,12 +7,12 @@ import org.oppia.android.app.model.PromotedStory
 // TODO(#297): Add download status information to promoted-story-card.
 
 /** [ViewModel] for displaying a promoted story. */
```

Comment:
Same as above, reccommend: `[ViewModel] for displaying a promoted story in the [RecentlyPlayedFragment].`

Reply:
done